apps/backend/api/services/Search/util.js

```diff
@@ -75,14 +75,17 @@ export const filterAndSortPosts = curry((opts, q) => {
     })
   }
 
-  if (afterTime) {
+  if (afterTime && beforeTime) {
+    q.where(q2 =>
+      q2.where('posts.start_time', '<', beforeTime)
+      .andWhere('posts.end_time', '>=', afterTime)
+    )
+  } else if (afterTime) {
     q.where(q2 =>
       q2.where('posts.start_time', '>=', afterTime)
```

Comment:
you're right, i think only needs to be end_time >= afterTime, dont need to worry about the start tike condition. unless... what if we have a post with a start time but no end time? i dont think we allow that right now though